libbeat/logp/metrics.go

```diff
@@ -71,6 +71,18 @@ var gauges = map[string]bool{
 	"beat.memstats.memory_total":     true,
 	"beat.memstats.memory_alloc":     true,
 	"beat.memstats.gc_next":          true,
+	"beat.cpu.usage":                 true,
+	"beat.cpu.usage.normalized":      true,
+	"beat.cpu.usage.since_start":     true,
+	"beat.info.uptime.ms":            true,
+	"host.cpu.usage":                 true,
+	"host.cpu.usage.normalized":      true,
+	"host.load_average.1m":           true,
```

Comment:
Let's follow this scheme here: https://github.com/elastic/beats/blob/master/metricbeat/module/system/load/_meta/data.json#L12